hikey.mk

```diff
@@ -1,12 +1,35 @@
+################################################################################
+# User-defined variables
+# Edit so these match your target
+# NOTE: If making changes after a build, please clean before rebuilding!
+################################################################################
+# Non-secure user mode (root fs binaries): 32 or 64-bit [default 64]
 NSU ?= 64
-SU ?= 32
+# Secure kernel (OP-TEE OS): 32 or 64-bit [default 64]
 SK ?= 64
+# Secure user mode (Trusted Apps): 32 or 64-bit [default 32, requires SK=64 for 64]
+SU ?= 32
+
+# Console UARTs: 3 or 0
```

Comment:
Talked to Victor about it, most common setup is most likely [sonic screwdriver](https://www.96boards.org/products/mezzanine/uarts/) (UART3), since the other option UART0, requires the 4-pin connector that has to be soldered and also you need the special 1.8V cable. Therefore we decided to use UART3 as the default. ARM-TF + OP-TEE will be tied to the SW flag, whilst EDK2 and LINUX will be tied to the NW flag.